[
  {
    "DefinitionID": "oval:com.ubuntu.bionic:def:2018205340000000",
    "Title": "CVE-2018-20534 on Ubuntu 18.04 LTS (bionic) - medium.",
    "Description": "** DISPUTED ** There is an illegal address access at ext/testcase.c in libsolv.a in libsolv through 0.7.2 that will cause a denial of service. NOTE: third parties dispute this issue stating that the issue affects the test suite and not the underlying library. It cannot be exploited in any real-world application. It was discovered that libsolv incorrectly handled certain malformed input. An attacker could use this issue to cause libsolv to crash, resulting in a denial of service.",
    "Advisory": {
      "Severity": "Medium",
      "Cves": [
        {
          "CveID": "CVE-2018-20534",
          "Cvss2": "",
          "Cvss3": "",
          "Cwe": "",
          "Impact": "",
          "Href": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-20534",
          "Public": ""
        }
      ],
      "Bugzillas": [],
      "AffectedCPEList": [],
      "Issued": "1000-01-01T00:00:00Z",
      "Updated": "1000-01-01T00:00:00Z"
    },
    "Debian": null,
    "AffectedPacks": [
      {
        "Name": "libsolv",
        "Version": "",
        "Arch": "",
        "NotFixedYet": true,
        "ModularityLabel": ""
      }
    ],
    "References": [
      {
        "Source": "CVE",
        "RefID": "CVE-2018-20534",
        "RefURL": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-20534"
      },
      {
        "Source": "Ref",
        "RefID": "",
        "RefURL": "http://people.canonical.com/~ubuntu-security/cve/2018/CVE-2018-20534.html"
      },
      {
        "Source": "Ref",
        "RefID": "",
        "RefURL": "https://bugzilla.redhat.com/show_bug.cgi?id=1652604"
      },
      {
        "Source": "Ref",
        "RefID": "",
        "RefURL": "https://github.com/openSUSE/libsolv/pull/291"
      },
      {
        "Source": "Ref",
        "RefID": "",
        "RefURL": "https://ubuntu.com/security/notices/USN-3916-1"
      }
    ]
  }
]
